src/pages/patientView/clinicalInformation/lib/clinicalAttributesStyleConsts.js

```diff
@@ -0,0 +1,285 @@
+/**
```

Comment:
This is a lot of Javascript to add to a codebase that is almost entirely Typescript. I take it we're doing JS here because it's being used by an existing JS file. Why not convert that file to Typescript and then make this typescript too? All this `@param {object}` documentation is killing me.

Don't change this unless Aaron agrees, because it's a fair bit of work.

Reply:
Ah, I figured to use the @param documentation after typing /** */ in VS Code, then it showed up behind the function I was defining